### DISCUSSION TITLE
Docker Single Image: How to find the generated .env file

### DISCUSSION THREAD

**Discussion by magickaito** [Budibase/budibase#6844] [Help] [answered]
Hi

I followed [Docker Single Image instructions](https://github.com/Budibase/budibase/tree/develop/hosting/single) to compile and run Budibase on DigitalOcean. 

How do I find the randomly generated couch DB credentials?

I tried `docker exec -it budibase /bin/bash` and look for the /data folder, couldnt see any .env file.
These are the files/folders I can see:
```
root@57e16e0723c7:/# ls
app   build-target-paths.sh  dev		   healthcheck.sh  lib	  minio   opt	run	   scripts  tmp  worker
bin   buildtarget.txt	     docker-entrypoint.sh  home		   lib64  mnt	  proc	runner.sh  srv	    usr
boot  data		     etc		   keys.asc	   media  nodejs  root	sbin	   sys	    var

root@57e16e0723c7:/# cd data
root@57e16e0723c7:/data# ls -a
.  ..  couch  minio
```

**Comment by mike12345567** [comment, marked as answer]
Hi @magickaito - just to check, what was the run command that you tried, and also which version of the single image are you using? The `.env` was only added to the most recent versions so it could be that.

**Reply by magickaito** [reply]
thanks! i forked at 1.1.7. I updated to 1.1.25 and I can see the .env file.
